policyengine_us/variables/gov/usda/snap/income/deductions/shelter/snap_standard_utility_allowance.py

```diff
@@ -10,10 +11,23 @@ class snap_standard_utility_allowance(Variable):
     definition_period = MONTH
 
     def formula(spm_unit, period, parameters):
-        utility = parameters(period).gov.usda.snap.income.deductions.utility
+        p = parameters(period).gov.usda.snap.income.deductions.utility.standard
         allowance_type = spm_unit("snap_utility_allowance_type", period)
         allowance_types = allowance_type.possible_values
         region = spm_unit.household("snap_utility_region_str", period)
+        spm_size = spm_unit("spm_unit_size", period)
+        MAX_SPM_SIZE = 9
```

Comment:
is this a policy parameter set by NC? If so we should make it a policy parameter

Reply:
What do you mean by policy parameter? I know that there are other states that also depend on household size, so we tried to build this to make it easier to add other states.

Reply:
I was more thinking about the case If we were to add e.g. Alabama which has a maximum size of 5 - this would not work I believe 
The easiest way might be to define a parameter with the max spm size for snap by state so we can just add it to the parameter

Reply:
NC only defines household sizes up to 5 as well. We just filled in the remaining household sizes with the same amount as defined for the 5 household size. Unless PE does [fancy indexing](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#:~:text=In%20this%20case%2C%20parameters(period).housing_benefit%5Bzone%5D%20would%20raise%20en%20error%2C%20whatever%20zone%20contains%2C%20as%20the%20homogeneity%20condition%20is%20not%20respected%3A%20housing_benefit.zone_1%20is%20a%20node%2C%20while%20housing_benefit.coeff_furnished%20is%20a%20parameter.) different than Open Fisca, you can not have different states with different household sizes defined.